### DISCUSSION TITLE
Co-Pilot not working in Visual Studio Codium

### DISCUSSION THREAD

**Discussion by jfscoertzen** [community/community#13641] [Copilot Conversations] [answered]
Co-pilot not working. Visual Studio Codium.
<img width="443" alt="Screenshot 2022-03-27 at 18 55 53" src="https://user-images.githubusercontent.com/2377987/160292247-bb301111-5adb-4abf-a45a-4414585eaa83.png">
<img width="562" alt="Screenshot 2022-03-27 at 18 58 45" src="https://user-images.githubusercontent.com/2377987/160292251-9c3e55c8-d7c0-4849-836d-0492a4ab2062.png">

**Comment by Xtendera** [comment]
Visual Studio Codium isn't officially supported. You have a better chance of getting an answer if you post it in [the VS Codium issues page](https://github.com/VSCodium/vscodium/issues).

**Comment by davecheney** [comment, marked as answer]
Sorry, this is a known issue that authentication does not work with VS Codium. Please use VScode.

**Reply by PinkaminaDianePie** [reply]
will it be fixed once `copilot` will be available to everyone? or it's EEE again?
